util/chplenv/chpl_atomics.py

```diff
@@ -51,7 +52,10 @@ def get(flag='target'):
             elif compiler_val in ['allinea', 'cray-prgenv-allinea']:
                 atomics_val = 'cstdlib'
             elif compiler_val == 'clang':
-                atomics_val = 'intrinsics'
+                if has_std_atomics(compiler_val):
+                    atomics_val = 'cstdlib'
+                else:
+                   atomics_val = 'intrinsics'
```

Comment:
You might check these last three lines to see if one of them has a tab in it.

Reply:
No tab, but spacing was wrong (missing a space) -- fixed